tox.ini

```diff
@@ -29,9 +29,13 @@ deps =
     py{27}: enum34
 
     # Python3.5+ only deps
-    py{35,36,37,38}: aiohttp >= 3.0.0
+    py{35,36,37,38}: aiohttp >= 3.3.1
```

Comment:
Have they deprecated support for aiohttp 3.0.0 ?

Reply:
I don't think so. But the aiobotocore setup file requires aiohttp >= 3.3.1 so I made that change too. I'm not sure if it has any implications.

Reply:
Okay!